staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go

```diff
@@ -141,18 +142,32 @@ func (c *healthzCheck) Check(r *http.Request) error {
 	return c.check(r)
 }
 
+// getExcludedChecks extracts the health check names to be excluded from the query param
```

Comment:
Where can we document this that users might see it?

Reply:
I don't actually have a good answer for this, I actually am unsure how people can learn about the verbose flag on the healthz endpoint currently without looking directly at the code.

Reply:
It should go into docs which describe how to set up kube-apiserver probes.  I don't think we have one, so maybe a comment at e.g. https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/manifests/kube-apiserver.manifest#L37 where I think we're going to use it.

It's not great that this is the canonical source today, but it's the reality.  Eventually that moves to the bundle, which I think it's more reasonable to say should act as a reference for anyone not using the bundle.

Reply:
I was planning on rolling out the manifest changes in a separate PR, is that okay or would you like to add that here? 

Insofar as the comment, I am happy to add a comment where ever it would make sense but that manifest file looks like JSON. I don't think JSON supports comments but it has been a while, so I could be mistaken. Unless you have something specific in mind (like adding a "_comment" key or something).

Reply:
Gah - you're right... remind me that we should express the bundle in yaml :-)

I was thinking that showing the health checks you're planning on using would make the intent much clearer here!